sunrise.py

```diff
@@ -0,0 +1,379 @@
+    """
+    B = frac_year(hour, leap_year)
+    et = 9.87 * np.sin(2 * B * (np.pi / 180)) - 7.53 * \
+        np.cos(B * (np.pi / 180)) - 1.5 * np.sin(B * (np.pi / 180))
+
+    return et
+
+
+def local_meridian(utc=utc):
+    """
+    This function calculates the local standard time meridian,
+    LSTM. The LSTM is a reference meridian for given time zones.
+    LSTM is calculated according to the equation:
+        LSTM = 360/24*UTC
+    UTC < 0 gives the answer in degrees West.
+    UTC > 0 gives the answer in degrees East.
+
+    Parameters:
+    -----------
+    utc : integer
+        UTC is the "coordinated universal time." This parameter
+        gives the UTC offset for a particular timezone.
+        E.g. Chicago time is UTC-5, so we take utc = 5.
+
+    Returns:
+    --------
+    lstm : float
+        The LSTM in degrees
+    """
+
+    lstm = 15 * utc
+
+    return lstm
+
+
+def time_correction(lstm, et, lon=lon):
+    """
+    The time correction factor adjusts for differences in
+    Local Solar Time (LST) and the location of interest, at
+    a given longitude. There is a factor of 4 due to the speed
+    of Earth's rotation.
+
+    Parameters:
+    -----------
+    lstm : float
+        The local standard time meridian.
+    et : float
+        The equation of time.
+    lon : float
+        The longitude of the location of interest.
+
+    Returns:
+    --------
+    tc : float
+        Time correction factor in minutes.
+    """
+    tc = 4 * (lon - lstm) + et
+
+    return tc
+
+
+def local_solar_time(local_time, tc):
+    """
+    Corrects the local time to give the local solar time.
+
+    Parameters:
+    -----------
+    local_time : float
+        The local time in hours
+    tc : the time correction factor
+
+    Returns:
+    --------
+    lst : float
+        The local solar time in hours.
+    """
+
+    lst = local_time + tc / 60
+
+    return lst
+
+
+def hour_angle(lst):
+    """
+    This function calculates the hour angle of the sun.
+    The hour angle is the number of degrees the sun has
+    moved across the sky.
+
+    Parameters:
+    -----------
+    lst : float
+        The local solar time
+
+    Returns:
+    --------
+    ha : float
+        The hour angle in minutes
+    """
+    ha = (15 * (lst - 12))
+
+    return ha
+
+
+def solar_elevation(ha, delta, lat=lat):
+    """
+    Calculate the solar elevation for a given hour angle,
+    latitude, and declination.
+
+    Parameters:
+    -----------
+    ha : float
+        The hour angle
+    delta : float
+        The declination angle
+    lat : float
+        The latitude at the location of interest.
+
+    Returns:
+    --------
+    alpha : float
+        The elevation angle
+    """
+
+    sin_term = np.sin(delta * np.pi / 180) * np.sin(lat * np.pi / 180)
+    cos_term = np.cos(delta * np.pi / 180) * np.cos(lat * \
+                      np.pi / 180) * np.cos(ha * np.pi / 180)
+    # alpha = np.arcsin((sin_term+cos_term)*np.pi/180)
```

Comment:
Is this comment supposed to still be here?

Reply:
In the future, if you think a change should be made, like removing an unused line, you can make a "code suggestion" during the review. 

Select the "+" next to the line you want to change. Then in formatting options you will find the first one looks like a "+/-". That will open a code suggestion. It will look like this: 
```suggestion
```